### ISSUE TITLE
Poor debug lines

### ISSUE THREAD

**Issue by smktpd** [adrienverge/openfortivpn#1250]
I am trying to debug why a connection to the new endpoint simply doesn't succeed and I run `sudo openfortivpn` with `-vvv` key and yet I get useless output:
```
sudo openfortivpn ${pub_host_of_vpn_server}:54443 --username=${my_user} -vvv
DEBUG:  ATTENTION: the output contains sensitive information such as the THE CLEAR TEXT PASSWORD.
DEBUG:  openfortivpn 1.22.1
DEBUG:  revision unavailable
DEBUG:  Loaded configuration file "/opt/homebrew/etc/openfortivpn/openfortivpn/config".
VPN account password:
DEBUG:  Configuration host = "${pub_host_of_vpn_server}"
DEBUG:  Configuration realm = ""
DEBUG:  Configuration port = "54443"
DEBUG:  Configuration username = "${my_user}"
DEBUG:  Configuration password = "${my_password}"
DEBUG:  Resolving gateway host ip
DEBUG:  Establishing TLS connection
DEBUG:  server_addr: ${pub_ip_of_vpn_server}
DEBUG:  server_port: 54443
DEBUG:  gateway_ip: ${pub_ip_of_vpn_server}
DEBUG:  gateway_port: 54443
```
and it just hangs there and nothing happens and this log doesn't let me figure out the reason.

**Comment by smktpd** [comment]
Nope, it was the opposite: the tunnel did not succeed, I got no IP.
In this specific case it turned out that Lulu (a firewall) after an update just didn't pop-up a prompt window and was blocking the connection by default.
But this ticket is about the lack of debug lines which could hint me why it fails.

**Comment by DimitriPapadopoulos** [comment]
I see. I guess openfortivpn calls a network function that is blocked by the firewall. The firewall doesn't implement any timeout so openfortivpn gets stucks forever. Isn't that a firewall issue? I mean, we won't implement a timeout in openfportivpn just to print a debug message.

**Comment by DimitriPapadopoulos** [comment]
Suggestion 1 looks OK.

As for 2, the message is probably too precise. Any program that tries to initiate a connection with an unreachable server freezes. It will eventually fail if the network function hits a system timeout. In any case, that's really up to the user to handle that. I've never seen software print such precise messages in such a case.
